uploader/pantheon.py

```diff
@@ -180,6 +197,9 @@ def process_file(path, filetype):
 
     Returns:
     list: It returns a list with value of the API call status_code and reason
+    :param path:
+    :param filetype:
+    :return:
```

Comment:
Is this part of some doc syntax that I'm not familiar with? Otherwise, it looks like these lines are unnecessary, since you describe the method above.

Reply:
those were generated by my editor. i will move those